index.html

```diff
@@ -3752,6 +3752,69 @@ <h3>Media Type Precision</h3>
           </p>
         </section>
 
+        <section>
+          <h2>JSON Processing</h2>
+
+          <p>
+While the base media types for this specification are JSON-LD documents, JSON-LD
+processing is not required to be performed in a variety of scenarios. These
+scenarios include, but are not limited to:
+          </p>
+
+          <ul>
+            <li>
+When using the JOSE or COSE specifications to secure a
+<a>verifiable credential</a> or <a>verifiable presentation</a>.
+            </li>
+            <li>
+When performing JSON Schema validation.
+            </li>
+            <li>
+When serializing or deserializing <a>verifiable credentials</a> or
+<a>verifiable presentations</a> into databases that index their contents.
+            </li>
+            <li>
+When operating on <a>verifiable credentials</a> or <a>verifiable
+presentations</a> in a software application after verification or validation
+is performed.
+            </li>
+            <li>
+When an application chooses to process the media type using the `+json`
+structured media type suffix.
+            </li>
+          </ul>
+
+          <p>
+That is, pure JSON processing is allowed as long as the document being
+consumed or produced is a <a>conforming document</a>. If pure JSON processing
+is desired, an implementer is advised to follow the following rule:
+          </p>
+
+          <ul>
+            <li>
+Ensure that all values associated with a `@context` property are in the
```

Comment:
```suggestion
consumed or produced is a <a>conforming document</a>. If pure JSON processing
is desired, an implementer MUST follow the following rule:
          </p>

          <ul>
            <li>
Ensure that all values associated with a `@context` property are in the
```

This section is normative. If this is indeed the intention, then this should be made by the usage of _MUST_.

If this is _not_ the case, then of course this text is o.k. as is, but the section must be marked as informative.

Reply:
That's fine, I wanted to start the PR out as purely informative and see if folks still desire normative statements on this. This statement was normative in v1.0 and v1.1. We'll see what the other reviewers say.

Reply:
@iherman wrote:
>  the section must be marked as informative.

Done in d4574f8da5c820c186ff2f87c5dd194ebffcab55.

Reply:
> Would it help to add a note here around the inclusion of `@vocab` into the core context making this a bit simpler?

Hmm, good point.

There are a number of things that we've done to make it easier to work w/ the data in a JSON-only mode. Use of `@protected`, use of `@vocab`, aliasing `@id` and `@type`, etc. We do talk about some of these mechanisms: 

* https://w3c.github.io/vc-data-model/#syntactic-sugar
* https://w3c.github.io/vc-data-model/#getting-started

... but as you note, we don't mention "the issuer defined vocabulary" anywhere in the spec, and we probably should. I've raised https://github.com/w3c/vc-data-model/issues/1210 to address this concern. We should probably add it in the "Syntactic Sugar" section.